### PR TITLE
Added how to configure InfluxDB Persistance via Web-UI

### PR DESCRIPTION
**Pull request by krikk**
added a paragraph about configuring InfluxDB Settings in Web-UI

### CODE REVIEW

File: bundles/org.openhab.persistence.influxdb/README.md [hunk 1]
```diff
@@ -34,6 +34,12 @@ This is very easy and you will find good documentation on it on the
 
 ## Configuration
 
+### Configuration via Web-Interface:
```
Comment (jlaur) [inline bundles/org.openhab.persistence.influxdb/README.md:37]:
```suggestion
### Configuration via web interface
```

Or perhaps use term "UI" like you did in header "### File based configuration (not needed when done via UI)" to not mix terms "UI" and "web interface"?

File: bundles/org.openhab.persistence.influxdb/README.md [hunk 2]
```diff
@@ -34,6 +34,12 @@ This is very easy and you will find good documentation on it on the
...
+All settings for this Persistance Service can be found under `Settings` → `Other Services` → `InfluxDB Persistance Service`
```
Comment (jlaur) [inline bundles/org.openhab.persistence.influxdb/README.md:38]:
```suggestion

All settings for this Persistence Service can be found under `Settings` → `Other Services` → `InfluxDB Persistence Service`
```

File: bundles/org.openhab.persistence.influxdb/README.md [hunk 3]
```diff
@@ -34,6 +34,12 @@ This is very easy and you will find good documentation on it on the
...
+
+Attention: The File based Configuration (described below) overrides the Configuration in the Web-Interface!
```
Comment (jlaur) [inline bundles/org.openhab.persistence.influxdb/README.md:40]:
```suggestion
Attention: The file-based configuration (described below) overrides web interface configuration.
```

File: bundles/org.openhab.persistence.influxdb/README.md [hunk 4]
```diff
@@ -34,6 +34,12 @@ This is very easy and you will find good documentation on it on the
...
+
+### File based configuration (not needed when done via UI)
```
Comment (jlaur) [inline bundles/org.openhab.persistence.influxdb/README.md:42]:
```suggestion
### File-based configuration
```

I would then suggest to move the sentence " (not needed when done via UI)" after next line, i.e.:
```
This service can be configured in the file `services/influxdb.cfg`.
This is not needed when configured in UI.
```

**Review by jlaur — CHANGES_REQUESTED** [review]
Thanks! I'm wondering if it would make sense to simplify this, by just adapting:

```
This service can be configured in the file `services/influxdb.cfg`.
```

to:

```
This service can be configured in the file `services/influxdb.cfg` or in the UI under `Settings` → `Other Services` → `InfluxDB Persistence Service`.
The file-based configuration overrides the UI configuration.
```

Would that be sufficient?

**Comment by krikk** [comment]
i would strongly suggest to put the UI approach first, because the config via UI is much more fool-proof than the file based approach (especially with influxDB2)

**Comment by jlaur** [comment]
@lujop - does it look fine to you?

File: bundles/org.openhab.persistence.influxdb/README.md [hunk 5]
```diff
@@ -34,7 +34,8 @@ This is very easy and you will find good documentation on it on the
 
 ## Configuration
 
-This service can be configured in the file `services/influxdb.cfg`.
+This service can be configured in the UI under `Settings` → `Other Services` → `InfluxDB Persistence Service` or in the file `services/influxdb.cfg`.
+Attention: The file-based configuration overrides the UI configuration!
```
Comment (jlaur) [inline bundles/org.openhab.persistence.influxdb/README.md:38]:
```suggestion
Attention: The file-based configuration overrides the UI configuration.
```

**Review by jlaur — APPROVED** [review]
Thanks!

**Pull request merged**